rusoto/core/Cargo.toml

```diff
@@ -42,6 +42,7 @@ time = "0.1.35"
 tokio = "0.1.7"
 tokio-timer = "0.2.6"
 xml-rs = "0.8"
+flate2 = "1.0"
```

Comment:
Can we change this dependency so it's only brought in when the `encoding` flag is active?

`flate2 = { version = "1.0", optional = true }`